pkg/provisioner/ironic/ironic.go

```diff
@@ -698,6 +699,13 @@ func (p *ironicProvisioner) setLiveIsoUpdateOptsForNode(ironicNode *nodes.Node,
 	updater.
 		SetInstanceInfoOpts(optValues, ironicNode).
 		SetTopLevelOpt("deploy_interface", "ramdisk", ironicNode.DeployInterface)
+
+	if p.config.liveISOForcePersistentBootDevice != "" {
+		driverOptValues := optionsData{
+			"force_persistent_boot_device": p.config.liveISOForcePersistentBootDevice,
+		}
+		updater.SetDriverInfoOpts(driverOptValues, ironicNode)
+	}
```

Comment:
You probably want an `else` branch with setting the default value or removing the existing value altogether.